shared/middleware/helpers/bucket_helper.rb

```diff
@@ -297,7 +297,8 @@ def restore_previous_version(encrypted_channel_id, filename, version_id, user_id
         )
         version_restored = true
         Honeybadger.notify(
-          "Restore at Specified Version Failed. Restored most recent.",
+          error_class: "#{self.class.name}Error",
+          error_message: "Restore at Specified Version Failed. Restored most recent.",
```

Comment:
Realized while updating here that, using the `error_class`, we can break apart errors that happen in `BucketHelper` by bucket.  You'll get error class names like `AnimationBucketError` and `SourceBucketError`, which will probably be useful for quickly diagnosing spikes in these events (not that you couldn't already figure that out by digging a little).